esp-hal/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed GPIO counts so that using async code with the higher GPIO number should no longer panic (#1361, #1362)
 - ESP32/ESP32-S2: Wait for I2S getting out of TX_IDLE when starting a transfer (#1375)
 - Fixed writes to SPI not flushing before attemting to write, causing corrupted writes (#1381)
+- Fixed TWAI DeadTimeCfg bit values (#1378)
```

Comment:
I think you meant MCPWM in these changelog entries right?

Reply:
Yes!